Sprint 21

hismram opened the pull request.

hismram committed:
ДЗ по спринту 21

hismram committed:
ДЗ по спринту 21

ArturNurtdinov reviewed (commented):
1. История поиска отображается без заголовка и кнопки "Очистить историю":
<img width="569" height="559" alt="image" src="https://github.com/user-attachments/assets/441b2732-fbdc-4a3f-a88f-54b67d01116a" />
Макет:
<img width="506" height="669" alt="image" src="https://github.com/user-attachments/assets/69005464-cd2f-447b-909b-df2ed3515435" />
2. Если удалить все избранные треки, то изображение-заглушка не отображается:
<img width="568" height="849" alt="image" src="https://github.com/user-attachments/assets/15225368-8271-49a9-8f19-eb9d8414c99a" />

hismram committed:
ДЗ по спринту 21
- Исправил заглушку пустого реестра избранных
- Вернул заголовок и кнопку очистки истории поиска

hismram committed:
ДЗ по спринту 21
- Исправил заглушку пустого реестра избранных
- Вернул заголовок и кнопку очистки истории поиска

hismram committed:
ДЗ по спринту 21
- Исправил заглушку пустого реестра избранных
- Вернул заголовок и кнопку очистки истории поиска
